README.md

```diff
@@ -8,7 +11,46 @@ A deployment of `BOPTEST-Service` is available at https://api.boptest.net
 
 # Getting Started as a User
 
+Clicking [this link](https://colab.research.google.com/github/NREL/boptest-service/blob/documentation_readme_changes/docs/Introduction_to_the_BOPTEST_framework.ipynb) will take you to an interactive tutorial that will help you get familiar with both core BOPTEST APIs and BOPTEST-service APIs. Have fun! 
+
+
 # Building and Running on a Personal Computer
+1) Download this repository: clone this repo to your preferred location with 
+
+ * ``git clone https://github.com/NREL/boptest-service.git {preferred_location}`` 
+ 
+  where `{preferred_location}` is where you want to clone the repo to. 
+  
+2) Install [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/).
+
+3) To build, use the following command within the root directory of the extracted software:
+
+  * ``docker-compose up web worker provision``
+
+4) In a separate process, use the core [BOPTEST APIs](https://github.com/ibpsa/project1-boptest/tree/boptest-service#test-case-restful-api) as well as [BOPTEST-service APIs](https://github.com/NREL/boptest-service/tree/documentation_readme_changes#boptest-service-specific-apis) to interact with the test case using your test controller.  
+Alternatively, view and run an example test controller as [described below](https://github.com/NREL/boptest-service#running-tests).
+5) Shutdown the test case by the command ``docker-compose down`` executed in the root directory of this repository.
+
+
+# BOPTEST-Service Specific APIs
+## APIs for the regular user 
+
+| Interaction                                                           | Request                                                   |
+|-----------------------------------------------------------------------|-----------------------------------------------------------|
+| List available test cases.                                |  GET `testcases` |
+| Check if specific test case is in list of available test cases. Returns status code of `200` (OK) if found, `404` if not found.                                 |  GET `testcases/{testcase_name}` |
+| Select a test case and begin a new test. A unique ``testid`` will be returned.                                |  POST ``testcases/{testcase_name}/select`` |
+| Stop a running test.                                                   |  PUT ``stop/{testid}`` |
```

Comment:
I have to add the [status API](https://github.com/ibpsa/project1-boptest/blob/bca18d7de86fe84d397117d5259435bb3233040a/web/server/src/routes/boptest.js#L98-L109), what would be a good description in the interaction column for it?